src/server/api/schema.js

```diff
@@ -1106,74 +1107,12 @@ const rootMutations = {
       { questionResponses, campaignContactId },
       { loaders, user }
     ) => {
-      const contact = await loaders.campaignContact.load(campaignContactId);
-      const campaign = await loaders.campaign.load(contact.campaign_id);
-      await assignmentRequiredOrAdminRole(
-        user,
-        campaign.organization_id,
-        contact.assignment_id,
-        contact
-      );
-
-      await cacheableData.questionResponse.save(
+      return UpdateQuestionResponses.updateQuestionResponses(
```

Comment:
I think my preference is that rather than change the function signature across the mutation file, line 1105 here should just be the imported function `updateQusetionResponses` without any ": async...." at all -- i.e. just the matched value exactly.

And then in the mutation file `updateQusetionResponses` should take the exact signature above.